app/src/main/java/fr/free/nrw/commons/db/AppDatabase.kt

```diff
@@ -0,0 +1,13 @@
+package fr.free.nrw.commons.db
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import fr.free.nrw.commons.contributions.Contribution
+import fr.free.nrw.commons.contributions.ContributionDao
+
```

Comment:
Add java docs.

Reply:
Do I add javadocs to classes I didn't make? I just converted the file

Reply:
It would be helpful towards our goal of having docs for all classes (although the person who originally had created the file should indeed have written it, I guess she missed this one).

Reply:
@misaochan kdoc added